### PR TITLE
Add points form

### PR DESCRIPTION
**Pull request by Sk1nnyB**

### CODE REVIEW

File: src/components/Data/data.js [hunk 1]
```diff
@@ -18,7 +19,7 @@ const StyledButton = styled(Button)`
 
 export const data = [
   {
-    cta: <StyledButton label="Add points" />,
+    cta: <Link to="/hpe-softcat-leaderboard/pointsform"><StyledButton label="Add points" /></Link>,
```
Comment (kyleThomas123) [inline src/components/Data/data.js]: Could we change this to submit sales as well
```suggestion
    cta: <Link to="/hpe-softcat-leaderboard/pointsform"><StyledButton label="Submit Sales" /></Link>,
```

File: src/pages/PointsForm.js [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+      }
+    }
+
+    try {
+      // Add data to Firestore
+      const docRef = doc(firestore, 'sales-forms', auth.currentUser.uid);
+      const randkey = createRandomString(25);
+      const docSnapshot = await getDoc(docRef);
+      if (docSnapshot.exists()) {
+        const res = await updateDoc(doc(firestore, 'sales-forms', auth.currentUser.uid), {[randkey]: formValues});
```
Comment (kyleThomas123) [inline src/pages/PointsForm.js:82]: What is res doing? Its throwing warnings which means it won't compile on firebase. If its needed can you add `// eslint-disable-next-line`
```
Line 81:15:  'res' is assigned a value but never used  no-unused-vars
  
Line 83:15:  'res' is assigned a value but never used  no-unused-vars
```

Comment (Sk1nnyB) [reply src/pages/PointsForm.js:82]: Will add the line, it isn't used but it is needed and could be used in future for further validation if needed.

File: src/pages/PointsForm.js [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+        break;
+      }
+    }
+
+    try {
+      // Add data to Firestore
+      const docRef = doc(firestore, 'sales-forms', auth.currentUser.uid);
+      const randkey = createRandomString(25);
+      const docSnapshot = await getDoc(docRef);
+      if (docSnapshot.exists()) {
```
Comment (Sk1nnyB) [inline src/pages/PointsForm.js:80]:
```suggestion
      if (docSnapshot.exists()) {
      // eslint-disable-next-line
```

**Review by kyleThomas123 — APPROVED** [review]

**Pull request merged**